$NetBSD: patch-grc_script_gnuradio-companion,v 1.3 2015/09/01 04:13:43 mef Exp $

Add message for NetBSD when failed to import gnuradio.
Or missing gnuradio-core package.

--- grc/scripts/gnuradio-companion.orig	2015-08-23 21:36:38.000000000 +0900
+++ grc/scripts/gnuradio-companion	2015-08-23 21:38:19.000000000 +0900
@@ -35,6 +35,7 @@ warnings.filterwarnings("always")
 
 GR_IMPORT_ERROR_MESSAGE = """\
 Cannot import gnuradio.
+You may be missing gnuradio-core package or
 
 Is the python path environment variable set correctly?
     All OS: PYTHONPATH
